src/plone/restapi/upgrades/ordering.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from persistent.list import PersistentList
+from zope.annotation.interfaces import IAnnotations
+from plone.folder.default import DefaultOrdering
+
+import six
+
+
+def ensure_child_ordering_object_ids_are_bytestrings(parent):
```

Comment:
`bytestrings` -> `native_strings`
I would also prefer `container` instead of `parent` (parent of what?)